packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -1,4 +1,7 @@
 import { ApiClient } from '../.'
+import BigNumber from 'bignumber.js'
+
+export const DEFAULT_FEE_RATE_IN_SATOSHIS = new BigNumber('10000')
```

Comment:
```suggestion
export const DEFAULT_FEE_RATE_IN_SATOSHI = new BigNumber('10000')
```

Reply:
I am not sure why did you you declare this const and export? 
Do you plan to use this somewhere other than current file?